notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>I suggest passing in the value of artifact_uri=MODEL_URI, as a Pipeline parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>I cannot, due to instance schema uri and prediction schema uri using the MODEL_URI. Pipeline parameter and string cannot be concatenated: "TypeError: unsupported operand type(s) for +: 'PipelineParam' and 'str'"</p>